pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -853,6 +855,7 @@ spec:
     {{- .TaskResult.readlistsvc.items | notFoundErr "target service not found" | saveIf "readlistsvc.notFoundErr" .TaskResult | noop -}}
     {{- jsonpath .JsonResult "{.items[*].spec.clusterIP}" | trim | saveAs "readlistsvc.clusterIP" .TaskResult | noop -}}
     {{- jsonpath .JsonResult "{.items[*].metadata.labels.openebs\\.io/persistent-volume-claim}" | default "" | trim | saveAs "readlistsvc.pvcName" .TaskResult | noop -}}
```

Comment:
how was this working previously, if `openebs.io/persistent-volume-claim` was never as label?

Reply:
This needs upgrade for older versions of openebs volumes